### DISCUSSION TITLE
Stationary obj

### DISCUSSION THREAD

**Discussion by phlnil** [blakeblackshear/frigate#12500] [General]
Hi, 
I surely need to go e bit deep on this behaviour :
Frigate works well ... but seems in trouble with cars:
with a stationary car (left for hours in the same exact position)
frigate sometimes identifies it as an event and marks with a red square its position.
nothing seems changed... (sometimes could be a bird... or shadow...)
Could be a good idea set a area exactly around that park lot to avoid these kind of false event ?
I am yet tring to keep quite simple the config... is this a issue ? or it never occur...
(today I got 15 events (11sec -21sec) : those are quite similar : the same clip) yestarday (some config) just 3 or 5.
(weather is the same)

Thank you!

**Comment by NickM-27** [comment]
For frigate 0.13 you should read https://docs.frigate.video/guides/parked_cars

As part of the new UI and improvements to stationary tracking this has been greatly improved in the next version (0.14)

**Reply by phlnil** [reply]
Thank you as always, 
yes I had found it and even I am putting a lot of wishes on the next release, 
I try to minimize the issues caused by the user (me) and learn better the limits of the system itself.
